packages/toolpad-app/src/toolpad/Home/index.tsx

```diff
@@ -557,7 +557,13 @@ export default function Home() {
         <Typography variant="h2">Apps</Typography>
         <CreateAppDialog open={createDialogOpen} onClose={() => setCreateDialogOpen(false)} />
         <Toolbar variant={'dense'} disableGutters sx={{ justifyContent: 'space-between' }}>
-          <Button onClick={() => setCreateDialogOpen(true)}>Create New</Button>
+          <Button
+            onClick={() => {
+              setCreateDialogOpen(true);
```

Comment:
is this intended?

Reply:
Nope, I changed it back to how it was before, was just trying some things earlier.